release/air_examples/dolly_v2_lightning_fsdp_finetuning/lightning-llm-finetuning-7b.ipynb

```diff
@@ -0,0 +1,1218 @@
+{
```

Comment:
for release tests, you can simply symlink to your example?

Reply:
Done!